autoload/lsp/omni.vim

```diff
@@ -264,6 +264,21 @@ function! s:get_completion_result(server_name, data) abort
     return {'matches': l:completion_result['items'], 'incomplete': l:completion_result['incomplete'] }
 endfunction
 
+function! s:sort_by_sorttext(i1, i2) abort
+    let l:text1 = get(a:i1, 'sortText')
+    let l:text2 = get(a:i2, 'sortText')
+
+    " sortText is possibly empty string
+    let l:text1 = l:text1 ? l:text1 : a:i1['label']
+    let l:text2 = l:text2 ? l:text2 : a:i2['label']
```

Comment:
[memo]
`falsy-operator` can use since `patch 8.2.1794`(maybe...).
Older version can not use, so use trinary operator.